Calibration/HcalCalibAlgos/macros/CalibCorr.C

```diff
@@ -144,6 +148,12 @@ unsigned int truncateId(unsigned int detId, int truncateFlag, bool debug = false
     //Ignore depth index for depth > 1 in HB and HE
     if (depth > 1)
       depth = 2;
+  } else if (truncate0 == 6) {
+    //Ignore depth index for depth > 1 in HB and HE
```

Comment:
The comment is the same as the one at [L148](https://github.com/cms-sw/cmssw/pull/45044/files#diff-d54af1ffe7c3b1c015cba727cd7149a6a3ca54a355ffdeb404196427970244beR148), but the range of `depth`'s is different: please fix the comment